src/test/java/ro/msg/learning/shop/controller/integrationTest/FailOrderCreatorRestControllerProfileTest.java

```diff
@@ -0,0 +1,270 @@
+package ro.msg.learning.shop.controller.integrationTest;
```

Comment:
please follow the naming conventions for packages

Reply:
Done - using lower case and "_"